src/main/java/rx/Completable.java

```diff
@@ -1971,19 +1972,40 @@ public final void subscribe(CompletableSubscriber s) {
         } catch (NullPointerException ex) {
             throw ex;
         } catch (Throwable ex) {
+            Exceptions.throwIfFatal(ex);
             ERROR_HANDLER.handleError(ex);
```

Comment:
No, if the exception is fatal, it skips the hook. Once the hook has been called, the exception is wrapped into a NullPointerException and thrown up, but then there is a catch (NullPointerException) to prevent it from calling the hook again.